src/editor/UI/EditorSurface.re

```diff
@@ -547,6 +550,20 @@ let createElement =
                 (),
               );
 
+              /* Draw configured rulers */
+              let renderRuler = ruler =>
+                Shapes.drawRect(
+                  ~transform,
+                  ~x=fst(bufferPositionToPixel(0, ruler)),
```

Comment:
Nice find on the `fst` method - I forget about this sometimes... 👍

Reply:
Thanks! 🙂